GPU/GPUTracking/Base/cuda/GPUReconstructionCUDA.cu

```diff
@@ -179,7 +179,13 @@ int GPUReconstructionCUDABackend::InitDevice_Runtime()
       GPUError("Could not set CUDA device!");
       return (1);
     }
-    if (GPUFailedMsgI(cuCtxCreate(&mInternals->CudaContext, 0, tmpDevice))) {
+    CUresult ctxerr = cuCtxCreate(&mInternals->CudaContext, 0, tmpDevice);
```

Comment:
Actually, I think it would be better to do something like:
if cuCtxCreate fails, print the error, then just continue
If cuMemGetInfo fails, print the warning, destroy the context, then just continue.

For cuCtxDestroy (the one that is already there, and the one to add), please use GPUFailedMsg instead of GPUFailedMsgI, because this would be a real error, so it is correct to just bail out.

Reply:
Updated the commit again. Not sure i agree to continue on every error, but shouldn't matter either way.